addons/sourcemod/scripting/shop/db.sp

```diff
@@ -2,7 +2,13 @@ Database h_db;
 
 ShopDBType db_type;
 
-DataPack backup_dp;
+enum struct backup_struct {
```

Comment:
I am not sure that `enum struct` is available on SM 1.9, 1.10

Reply:
You have already upgraded to 1.10+.
https://github.com/FD-Forks/Shop-Core/blob/b4d2a9d051d103d72fbcc936f7eca3792ac75a0c/addons/sourcemod/scripting/shop/admin.sp#L7-L23